test/test_nn.py

```diff
@@ -8174,9 +8174,9 @@ def test_batchnorm_large_batch(self, device, dtype):
     @dtypes(torch.float, torch.double, torch.bfloat16, torch.complex128)
     def test_conv_empty_input(self, device, dtype):
         def help(input, conv, memory_format):
-            ref_out = conv(input)
+            ref_out = conv(input).detach()
```

Comment:
These `.detach()` calls are to ensure the autograd graph is not alive during `.to()` which will increase the `refcount` of params and prevent `swap_tensors` from being used.

As discussed, this was a known limitation of the `swap_tensors` path, but is unlikely to occur in practice (you don't normally want to change the dtype/device of your model after running forward).

Reply:
yea I agree that in real life it is unlikely to happen